samples/Bluetooth/BluetoothLESample3/BluetoothLESample3.nfproj

```diff
@@ -33,9 +33,14 @@
       <HintPath>packages\nanoFramework.CoreLibrary.1.12.0\lib\mscorlib.dll</HintPath>
       <Private>True</Private>
     </Reference>
+<<<<<<< Updated upstream
```

Comment:
Something went wrong with the merge... 😯

Reply:
Don't know what wrong. Thought it was working project. Fixed now.